Validation/MuonME0Validation/plugins/MuonME0SegHarvestor.cc

```diff
@@ -0,0 +1,161 @@
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "TTree.h"
+#include "TFile.h"
```

Comment:
@calabria - TTree and TFile includes are not needed (here and elsewhere in the PR)

Otherwise, please convert Int_t and Double_t to native types. Thanks